scripts/miniapps/HPGMG.md

```diff
@@ -0,0 +1,112 @@
+# Instruction for building and running HPGMG
+
+Implementation: C, MPI+OpenMP, uses MPI_THREAD_FUNNELED
+
+Useful links:
+- HPGMG website: http://crd.lbl.gov/departments/computer-science/PAR/research/hpgmg/
+- Bitbucket repository: https://bitbucket.org/hpgmg/hpgmg/
+
+## How to build
+
+1. Download from the Bitbucket repository
+   ```
+% git clone https://bitbucket.org/hpgmg/hpgmg/
+   ```
+   > git clone 
+
+   README.md original instructions which are slightly obsolete and
+   incomplete.
+   
```

Comment:
This is the README.md in the hpgmg repo.  But I see a type-o here that I will fix.  Hopefully that will make it clearer.